assets/tests/components/__snapshots__/ladderPage.test.tsx.snap

```diff
@@ -27,12 +27,27 @@ exports[`LadderPage renders the empty state 1`] = `
         />
       </button>
     </div>
+    <div
```

Comment:
**thought:** Ideally, when a snapshot test changes, you can read the snapshot, understand the changes, and verify that they're what you want. This file has 1400 lines of changes, so I, uh, have not reviewed it.

Perhaps there are other ways to rewrite the tests so they explicitly make a check, instead of depending on manually reading the snapshot. For example, "renders with selectedRoutes in different order than routes data" could query for route headers, grab the route names from them, and assert that the route names show up in the right order.

That's shouldn't be part of this PR though.